src/unit_test.c

```diff
@@ -128,7 +128,7 @@ void benchmark_storage_number(int loop, int multiplier) {
 		for(i = 0; i < loop ;i++) {
 			n *= multiplier;
 			if(n > STORAGE_NUMBER_POSITIVE_MAX) n = STORAGE_NUMBER_POSITIVE_MIN;
-			snprintf(buffer, 100, CALCULATED_NUMBER_FORMAT, n);
+			snprintfz(buffer, 100, CALCULATED_NUMBER_FORMAT, n);
```

Comment:
This must be 99 (again, the number will never go that far, but the right is 99 - or change the allocation to be 101)